shared/src/business/entities/EntityValidationConstants.ts

```diff
@@ -442,7 +442,6 @@ export const DOCKET_ENTRY_VALIDATION_RULE_KEYS = {
     otherwise: joi.optional(),
     then: joi.required(),
   }),
-  workItem: joi.object().optional(),
```

Comment:
I know the new fields are optional, so I can see both sides: is it worth adding `qcComplete`, `qcViewed` and/or `workItemId` the the joi validation?